helm/istiod/values.yaml

```diff
@@ -221,6 +221,9 @@ meshConfig:
   # No hurry to do this in 1.6, we're trying to prove the code.
 
 global:
+  # whether to use autoscaling/v2 template for HPA settings
+  # for internal usage only, not to be configured by users.
+  autoscalingv2API: true
```

Comment:
Can we hide it? Can we determine which version should be used based-on k8s cluster version?